lib/Sema/TypeCheckCompilerEvaluable.cpp

```diff
@@ -26,26 +26,205 @@ using namespace swift;
 
 namespace {
 
-/// Checks that the body of a function is compiler evaluable.
-/// Currently a skeleton implementation that only rejects while loops.
+/// Checks that a type is compiler representable.
+/// Currently a skeleton implementation that only rejects types named Float,
+/// Double and String.
 /// TODO(marcrasi): Fill in a real implementation.
+static bool checkCompilerRepresentable(const Type &type) {
+  return type.getString() != "Double" && type.getString() != "Float" &&
```

Comment:
Err, this is very very stubbed out... if this is what you want for this patch, please make this the subject of your next patch :-)

Reply:
will do